etc/schema/tasks.yaml

```diff
@@ -65,3 +65,7 @@ tasks:
       allow_limit:
         type: boolean
         required: True
+      tags:
+        type: list
+        required: True
```

Comment:
I don't think it should be required. My expectation is that it should be used rarely.